pkgs/misc/screensavers/multilockscreen/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv, makeWrapper, fetchFromGitHub, writeShellScriptBin,
+  imagemagick, i3lock-color, xdpyinfo, xrandr, bc, feh, procps, xrdb, xset,
+  gnused, gnugrep, coreutils
+}:
+let
+  i3lock = writeShellScriptBin "i3lock" ''
+    ${i3lock-color}/bin/i3lock-color "$@"
+  '';
+  binPath = stdenv.lib.makeBinPath [
+    imagemagick i3lock
+    xdpyinfo xrandr xset
+    bc feh procps xrdb
+    gnused gnugrep coreutils
+  ];
+in
+stdenv.mkDerivation rec {
+  pname = "multilockscreen";
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jeffmhubbard";
+    repo = "multilockscreen";
+    rev = "869e17b134a3687e3147fa06a051b052820ece87";
```

Comment:
Does upstream tag their releases? Can you reference a tag here instead of a commit sha?

Reply:
When I created this derivation, there were no tags.
Looks like there is now a single tag for `v1.0.0`, so I have updated the derivation to use that.